bootstrap-stage2.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+set -e # Exit on any error
+
+PREFIX=$PWD/bootstrap
+
+if [ "$OS" = "windows" ]; then
+    # IDRIS_PREFIX is only used to build IDRIS2_BOOT_PATH
+    IDRIS_PREFIX=$(cygpath -m "$PREFIX")
+    SEP=";"
+else
+    IDRIS_PREFIX=$PREFIX
+    SEP=":"
+fi
+
+# Default to chez CG
+if [ "$1" = "" ]; then
+    IDRIS2_CG="chez"
+else
+    IDRIS2_CG=$1
+fi
```

Comment:
How about `IDRIS2_CG ?= "chez"`? i.e. only override the variable if it's not set rather than
relying on an argument being passed to the script.